Resources/Prototypes/Entities/Mobs/Player/observer.yml

```
@@ -71,6 +71,19 @@
   id: MobObserver
   components:
   - type: Spectral
+  - type: IntrinsicUI # Mono
+    uis:
+      enum.RadarConsoleUiKey.Key:
+        toggleAction: ActionAGhostShowRadar
+  - type: RadarConsole # Mono
+    followEntity: true
+    maxRange: 512
+  - type: UserInterface # Mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)